.teamcity/_self/projects/WPComPlugins.kt

```diff
@@ -22,14 +22,6 @@ object WPComPlugins : Project({
 	params {
 		param("docker_image", "registry.a8c.com/calypso/ci-wpcom:latest")
 		param("build.prefix", "1")
-		checkbox(
```

Comment:
We don't need this checkbox to apply to the other builds, like the source map uploader.